client/src/Routes.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { List } from "semantic-ui-react";
+import Home from "./components/Home";
+import About from "./components/About";
+import Status from "./components/Status";
+
+const Routes = () => {
+  return (
+    <Router>
+      <div>
+        <List link>
+          <List.Item to="/" active as={Link}>
```

Comment:
The active props should be set on the active link only. So if the user is currently on the home page, the home link is active. If the user is on the /about page, the About link should be active instead and so on.